### PR TITLE
[CU-8686r6yae] Clubs feedback 

### PR DESCRIPTION
**Pull request by lrojas94**

### CODE REVIEW

**Comment by JustMaier** [comment]
Task linked: [CU-8686r6yae Club Launch Bugs (in Prod)](https://app.clickup.com/t/8686r6yae)

File: src/components/Club/club.utils.ts [hunk 1]
```diff
@@ -95,6 +95,28 @@ export const useMutateClub = () => {
...
 
+  const deleteClubTierMutation = trpc.club.deleteTier.useMutation({
+    async onSuccess() {
+      await queryUtils.club.getTiers.invalidate();
+    },
+    onError(error) {
+      try {
+        // If failed in the FE - TRPC error is a JSON string that contains an array of errors.
+        const parsedError = JSON.parse(error.message);
+        showErrorNotification({
```
Comment (manuelurenah) [inline src/components/Club/club.utils.ts:106]: Might be good to this on the server side instead

File: src/server/services/club.service.ts [hunk 2]
```diff
@@ -465,6 +684,8 @@ export const upsertClubResource = async ({
   const clubIds = clubs.map((c) => c.clubId);
   const contributingClubs = await userContributingClubs({ userId, clubIds });
 
+  console.log(clubIds, contributingClubs, userId);
```
Comment (manuelurenah) [inline src/server/services/club.service.ts:687]: Leftover
```suggestion
```

File: src/server/services/clubMembership.service.ts [hunk 3]
```diff
@@ -360,7 +367,7 @@ export const completeClubMembershipCharge = async ({
         clubMembershipChargeId: clubMembershipCharge.id,
         stripePaymentIntentId: clubMembershipCharge.invoiceId,
       },
-      externalTransactionId: `club-membership-charge-${clubMembershipCharge.id}`,
+      externalTransactionId: `club-membership-chrge-${clubMembershipCharge.id}`,
```
Comment (manuelurenah) [inline src/server/services/clubMembership.service.ts:370]: Might be a mistake?

**Review by manuelurenah — APPROVED** [review]

**Pull request merged**